README.md

```diff
@@ -595,6 +585,16 @@ Where `<docker_run>` will be non-empty if this particular task specified a Docke
 docker run -v <local_workflow_dir>:/root/<workflow_uuid> -i <image>
 ```
 
+> **NOTE**: If you are using the local backend with Docker and Docker Machine on Mac OS X, by default Cromwell can only
+> run from in any path under your home directory.
+>
+> The `-v` flag will only work if `<local_workflow_dir>` is within your home directory because VirtualBox with
+> Docker Machine only exposes the home directory by default.  Any local path used in `-v` that is not within the user's
+> home directory will silently be interpreted as references to paths on the VirtualBox VM.  This can manifest in
+> Cromwell as tasks failing for odd reasons (like missing RC file)
+>
+> See https://docs.docker.com/engine/userguide/dockervolumes/ for more information on volume mounting in Docker.
+
```

Comment:
I think there may be rebase issues here, this was Scott's work on a separate branch